scalafix-tests/unit/src/test/scala-target2/scalafix/tests/cli/BaseCliSuite.scala

```diff
@@ -159,10 +159,16 @@ trait BaseCliSuite extends AnyFunSuite with DiffAssertions {
     )
   }
 
+  // The defaults of this helper expect to find test input which is currently
+  // built only on scala 2.x since building them for Scala 3 would trigger
+  // test failures as the rules they exercise are not supported on Scala 3.
+  // TODO: switch defaults to rely on much more simple test input/rules so
+  // that we don't need to hardcode a scala version here.
+  val sourceDir = "scalafix-tests/input/src/main/scala-2/"
```

Comment:
only one user of this helper is actively relying on defaults and fails on Scala 3 targets, but I moved all of them for consistency